# goal_range: 1.5

# angle_min: -70
# angle_max: 70
# angle_increment: 1
# span_angle: 10
# goal_risk_range: 10

# ob_exist_coefficient: 0
# ob_near_coefficient: 0
# goal_orient_coefficient: 7

# kp_angle : 0.45
# ki_angle : 0.01 # max 0.05?
# kd_angle : 0.1

# # kp_distance : 40
# # ki_distance : 40
# # kd_distance : 40

# servo_middle: 93
# servo_left_max: 118 #93+25
# servo_right_max: 68 #93-25

# thruster_max : 1850
# thruster_min : 1600
# thruster_average : 1650

#------------------------

goal_range: 1.5
ob_dist_range : 10 #m
ob_angle_range : [-90, 90]
servo_range : [68, 118]
thruster_speed : 1650
span_angle : 5
filter_queue_size : 5
angle_alpha : 1